test/wallet/wallet_suite_test.go

```diff
@@ -99,7 +99,7 @@ var _ = BeforeEach(func() {
 })
 
 var allPassed = true
-var currentVersion = "3.1.0"
+var currentVersion = "3.1.1"
```

Comment:
BTW I already tagged 3.1.1 we could either re-tag it or make it 3.1.2

Reply:
ah yeah, just noticed it, @mischat  is the right person to talk about tagging, I am not opinionated.

Reply:
on the tagging, can we please stop tagging things and have conversations about them. The tags are basically within our bug bounty, and as a result, i don't want to just tag random things. It is OK to get a version in the code, and to tag the actual repo when we are happy with the state of master.

In short, let's have conversations before we tag things, i thought you updated the currentVersion variable in the repo (which i saw) but I didn't now that you actually tagged it. @nostdm

Reply:
OK, I thought this applied to minor/major versions and not the patch versions